features/steps/laravel_steps.rb

```diff
@@ -13,15 +13,29 @@
 end
 
 When(/^I wait for the app to respond on the appropriate port$/) do
-  laravel_port = ENV['LARAVEL_FIXTURE'] === 'laravel58' ? 61258 : 61256
   steps %{
-    When I wait for the app to respond on port "#{laravel_port}"
+    When I wait for the app to respond on port "#{getLaravelPort}"
   }
 end
 
 When("I navigate to the route {string}") do |route|
-  laravel_port = ENV['LARAVEL_FIXTURE'] === 'laravel58' ? 61258 : 61256
   steps %{
-    When I navigate to the route "#{route}" on port "#{laravel_port}"
+    When I navigate to the route "#{route}" on port "#{getLaravelPort}"
   }
+end
+
+Then("the exception {string} matches one of the following:") do |path, values|
```

Comment:
This step has been added to facilitate the change mentioned in [this comment](https://github.com/bugsnag/bugsnag-laravel/pull/374/files#r356519382), as Maze-runner v1 doesn't have any steps for multi-string checking.  As mentioned in the linked comment, this step could be replaced with a more specific step verifying the `errorClass` is correct for the given Laravel version.